projects/dvl-fw/src/lib/shared/project-serializer-service.ts

```diff
@@ -20,15 +21,17 @@ export class ProjectSerializerService {
     this.registry = ProjectSerializer.defaultRegistry;
   }
 
-  createProject(template: 'isi' | 'nsf' | 'csv' | 'json', fileContents: string, fileName?: string): Observable<Project> {
+  createProject(template: 'isi' | 'nsf' | 'csv' | 'json',
+    fileContents: string[] | string, fileNames?: string[] | string): Observable<Project> {
+    fileNames = isArray(fileNames) ? fileNames : [fileNames];
```

Comment:
Need to also take into account when fileNames is null...
`fileNames = isArray(fileNames) || !fileNames ? fileNames : [fileNames];`